src/class-parsely.php

```diff
@@ -41,6 +41,26 @@
  *   plugin_version: string,
  * }
  *
+ * @phpstan-type WP_HTTP_Request_Args array{
```

Comment:
1. I think we shouldn't have `WP_` in interface because this indicates that the type is defined via WordPress but in actual it is not.

2. Maybe we should only define the keys that we are using in our codebase.

Reply:
> I think we shouldn't have WP_ in interface because this indicates that the type is defined via WordPress but in actual it is not.
This is an exact copy of WP_Http::request args, so this is why `WP` was included. But I can remove it if you feel strongly about it.

Reply:
> Maybe we should only define the keys that we are using in our codebase.

Removing the keys works as well, but did as you recommended.

Reply:
> But I can remove it if you feel strongly about it.

No strong preferences just mentioning it because it seems like we are breaking a convention.

Reply:
OK, got it. Will leave as is for now because IMO it makes sense to have it, but if we see that this confuses us at any point we'll remove the `WP_` prefix.